### PR TITLE
Fixes #17: Implemented basic word counter & test

### LINKED ISSUE #17: Implement word count feature for the contents of the blog post

### PR DESCRIPTION
**Pull request by birtony**
Fixes #17: This function utilizes [wordcount](https://www.npmjs.com/package/wordcount) as suggested by @humphd in https://github.com/Seneca-CDOT/telescope/issues/17#issuecomment-549649707

### CODE REVIEW

**Comment by agarcia-caicedo** [comment]
I noticed you have these eslint errors, could you fix them so I can review the code again?
```
C:\Users\Deep Thought\Documents\GitHub\telescope\src\word-counter.js
  3:23  warning  Unexpected unnamed async function             func-names
  3:37  error    Missing space before function parentheses     space-before-function-paren
  4:1   error    Expected indentation of 2 spaces but found 4  indent

C:\Users\Deep Thought\Documents\GitHub\telescope\test\word-counter.test.js
  3:68  error  Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`  arrow-body-style
  4:1   error  Expected indentation of 2 spaces but found 4                                                           indent
  4:56  error  Expected parentheses around arrow function argument                                                    arrow-parens
  5:1   error  Expected indentation of 4 spaces but found 8                                                           indent
  6:1   error  Expected indentation of 2 spaces but found 4                                                           indent
```

**Comment by cyh0968** [comment]
I think it may be good to integrate the basic features as a module since reading-time, word-count, the writing level is closely connected.

**Comment by humphd** [comment]
@birtony you don't want to merge `master` here.  You need to rebase.  Also, you are missing a comma in your package.json file now.

**Comment by birtony** [comment]
@humphd thanks for the quick correction! Should I just drop the merge commit now and then rebase?

**Comment by humphd** [comment]
Yeah, get rid of that merge commit and rebase with master after you've updated it.

**Comment by birtony** [comment]
Alright! Just to clarify, should I use `git rebase -p --onto SHA^ SHA`, where `SHA` is the hash of the merge commit?

**Comment by humphd** [comment]
You can try it and see if it's what you want, sure.  Otherwise, you can do:

```
git checkout -B issue#17 68809a19e40f84432d4167a5c098911087cae46c
git checkout master
git pull upstream master
git checkout issue#17
git rebase master
npm install
# fix any issues in your code ...
git push origin issue#17 -f
```

**Comment by birtony** [comment]
Got it, thank you!

**Comment by humphd** [comment]
@birtony I don't think your lock file is valid.  Can you make sure you run `npm install` and add the `package-lock.json` file to this?  Travis can't build because it's missing dependencies.

**Comment by birtony** [comment]
My bad, haven't run `npm ci`...

**Comment by birtony** [comment]
I was not sure if we are including `package-lock.json` to the commit as it's auto generated?

**Comment by humphd** [comment]
`ci` is like `install`, it just does a clean install.  If the deps in the package.json don't match the lock file, it will die.  If you touch package.json, which you have, you need to also include package-lock.json in your commit.

**Comment by humphd** [comment]
OK, this looks better.  That failure is #117, unrelated to your PR.

**Comment by birtony** [comment]
I have almost decided to get sad

**Comment by humphd** [comment]
In https://github.com/Seneca-CDOT/telescope/pull/78#issuecomment-552089989 above, there's a suggestion to fold this into the work on reading level.  @cyh0968, I'd like to land this code, then rework the reading level code to include this test, and replace the implementation.

**Comment by birtony** [comment]
> In [#78 (comment)](https://github.com/Seneca-CDOT/telescope/pull/78#issuecomment-552089989) above, there's a suggestion to fold this into the work on reading level. @cyh0968, I'd like to land this code, then rework the reading level code to include this test, and replace the implementation.

Should I file it as another issue then?

**Comment by humphd** [comment]
No, we'll do it in #78.

**Comment by birtony** [comment]
Sure

**Comment by UltimaBGD** [comment]
Before it can be merged though, the comment requesting changes would not be made stale as it does not reference any code that has been updated. If all is fixed, we either need the commenter to approve themselves, or have someone dismiss that change.

**Comment by humphd** [comment]
You have conflicts in the `package*` files.  You have to merge quick when the tree is clean, or your code will get stale!

**Comment by UltimaBGD** [comment]
I don't think he would be able to merge until the change requested is gone, but I'm not sure.

**Comment by birtony** [comment]
How can I merge if @agarcia-caicedo doesn't review the changes?

File: package.json [hunk 1]
```diff
@@ -34,6 +34,8 @@
     "express": "^4.17.1",
     "express-healthcheck": "^0.1.0",
     "feedparser-promised": "^2.0.1",
+    "sentiment": "^5.0.2",
+    "wordcount": "^1.1.1"
```
Comment (humphd) [inline package.json]: This will break, you are missing a `,` here

Comment (birtony) [reply package.json]: Sorry, my fault. I have updated `package.json` now and rebased.

**Comment by birtony** [comment]
Hey @agarcia-caicedo, could you, please, re-review the requested change?

**Comment by birtony** [comment]
I am not sure what causes this...
After doing the following:
1. git checkout -B issue#17 7d0fc90
2. git checkout master
3. git pull upstream master
4. git checkout issue#17
5. git rebase master
6. npm install

No new changes are introduced to `package-lock.json`, `git status` says the working tree is clean. However, `npm test` is still failing. It cannot resolve the path to a number of modules, which are, in fact, included into both `package.json` and `package-lock.json`.

**Comment by humphd** [comment]
`Unexpected end of JSON input`, `SyntaxError: /Users/humphd/repos/telescope/package-lock.json: Unexpected end of JSON input`

So this package-lock.json file is malformed.  Delete it before you call `npm install`, and try again.

**Comment by birtony** [comment]
Alright, it should finally ge good now. Thank you, @humphd!

**Comment by humphd** [comment]
@birtony I'll help get this landed manually.  It's going to be a pain due to the rapid changes to the packages, and some errors in how previous commits did it.

**Comment by humphd** [comment]
OK, this is merged!  That wasn't easy :)

**Comment by birtony** [comment]
I appreciate your help, thank you, @humphd!

**Pull request merged**